daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus-queues.md

```diff
@@ -1,14 +1,17 @@
 ---
 type: docs
-title: "Azure Service Bus"
-linkTitle: "Azure Service Bus"
-description: "Detailed documentation on the Azure Service Bus pubsub component"
+title: "Azure Service Bus Queues"
+linkTitle: "Azure Service Bus Queues"
+description: "Detailed documentation on the Azure Service Bus Queues pubsub component"
 aliases:
-  - "/operations/components/setup-pubsub/supported-pubsub/setup-azure-servicebus/"
+  - "/operations/components/setup-pubsub/supported-pubsub/setup-azure-servicebus-queues/"
 ---
 
 ## Component format
-To setup Azure Service Bus pubsub create a component of type `pubsub.azure.servicebus`. See [this guide]({{< ref "howto-publish-subscribe.md#step-1-setup-the-pubsub-component" >}}) on how to create and apply a pubsub configuration.
+
+To setup Azure Service Bus Queues pubsub create a component of type `pubsub.azure.servicebus.queues`. See [this guide]({{< ref "howto-publish-subscribe.md#step-1-setup-the-pubsub-component" >}}) on how to create and apply a pubsub configuration.
```

Comment:
I would be useful to indicate any benefits/differences between using ASB Queues over ASB Topics. Why would you choose one over the other? Maybe it is just including this link https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-queues-topics-subscriptions as a reference. Given Queues in new in this release, some overview here would be valuable IMO. WDYT?

Reply:
The link was actually already there, on the next line! But I made it more clear that it explains the difference